lib/src/us/kbase/fbatools/PredictAuxotrophyParams.java

```diff
@@ -2,6 +2,7 @@
 package us.kbase.fbatools;
```

Comment:
Where did you get these changes from?

Reply:
@bio-boris that was just make from latest sdk